jekyll-linkpreview.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
   end
   spec.require_paths = ["lib"]
 
-  spec.add_dependency "jekyll", ">= 3.5", "< 4.1"
+  spec.add_dependency "jekyll", ">= 3.5"
```

Comment:
```suggestion
  spec.add_dependency "jekyll", ">= 3.5", "< 5.0"
```

Reply:
sounds like the right solution. thanks @ysk24ok